image_transport/package.xml

```diff
@@ -1,4 +1,4 @@
-<package>
+<package format="2">
```

Comment:
We should update maintainer names to avoid original maintainers receiving email when we start testing / releasing this on the ROS 2 farm

Reply:
Done.